### ISSUE TITLE
Can't clone: permission denied 

### ISSUE THREAD

**Issue by miguelpaz** [mrmrs/jkl-tachyons#4]
I tried to figure out what I did wrong before opening an issue but when I try to clone:
`git clone git@github.com:mmrs/jkl-tachyons.git`
I get this message:
Cloning into 'jkl-tachyons'...
Permission denied (publickey).
fatal: Could not read from remote repository.
Please make sure you have the correct access rights
and the repository exists.

**Comment by mrmrs** [comment]
@miguelpaz are you able to clone any other repos successfully?

**Comment by miguelpaz** [comment]
Yes, I don´t have problems cloning other repos. That´s why I opened the issue :)

**Comment by pschfr** [comment]
FWIW I get the same thing 😞 
![capture](https://user-images.githubusercontent.com/490222/27641139-073b03dc-5be9-11e7-9a70-06b6a0ba5da2.PNG)

**Comment by ericcarraway** [comment]
I initially ran into the same error message, but was able to spin up the project with the following:

**Terminal 1:**
```
git clone https://github.com/tachyons-css/jekyll-tachyons.git

cd jekyll-tachyons/

npm install

npm start
```

**Terminal 2:**
```
cd jekyll-tachyons/

jekyll serve --watch
```

**Browser:**
```
http://127.0.0.1:4000/
```

The Git URL may be out of date.  @briankerr addresses this in PR https://github.com/tachyons-css/jekyll-tachyons/pull/5.

Hope this helps.  👍

**Comment by pschfr** [comment]
@ericcarraway works for me!

**Comment by ghost** [comment]
You have to change the url from git@github.com:mmrs/jkl-tachyons.git (ssh) to https://github.com/tachyons-css/jekyll-tachyons (https).
So you'd do this:
`git clone https://github.com/tachyons-css/jekyll-tachyons`
